jans-cedarling/cedarling/examples/log_init.rs

```diff
@@ -40,7 +40,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     println!("Authz initialized with log type: {:?}", log_type);
 
     // Create the Authz instance with the selected log type
```

Comment:
This comment might need replacement or removal.

Reply:
thanks, fixed